packages/shared/src/components/icons/Bookmark/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Icon, { Size } from '../../Icon';
+import OutlinedIcon from './outlined.svg';
+import FilledIcon from './filled.svg';
+
+type Props = {
```

Comment:
I think we should create an interface that will be shared across the Icon components. It is most likely the shape of their props are the same.